dev-python/node-semver/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
```

Comment:
ops. I started from a package's metadata.xml.
I added the node-semver because is required by conan 0.16.0

My final task is to porting plex media player 1.2 that requires the last conan. (Furthermore I cannot proceed anyway because plex media player requires qt 5.7.1)